bin/sapconf

```diff
@@ -184,7 +184,7 @@ case "$1" in
       exit 1
     fi
     profile=${profile:-sapconf-hana}
```

Comment:
we need to change line 182 to check only for 'sapconf' profiles
if [[ $profile != "" && $profile == sapconf-* ]]; then
because only then a 'stop' action makes sense.

and we need to change the profile setting to an absolute setting
profile=sapconf-hana
because the use of 'sapconf ' is exactly to set a defined 'sapconf' profile.